types/src/error.rs

```diff
@@ -172,13 +176,29 @@ pub enum Error {
     #[error("Data square index out of range. row: {0}, column: {1}")]
     EdsIndexOutOfRange(u16, u16),
 
-    /// Could not create EDS, provided number of shares doesn't form a pefect square
+    /// Could not create EDS, provided number of shares doesn't form a pefect square.
     #[error("Invalid dimensions of EDS")]
     EdsInvalidDimentions,
 
     /// Zero block height.
     #[error("Invalid zero block height")]
     ZeroBlockHeight,
+
+    /// Share is not a start of a new blob.
+    #[error("Share is not sequence start")]
+    NotSequenceStart,
```

Comment:
```suggestion
    /// Share is not a start of a new blob.
    #[error("Expected sequence starting share")]
    ExpectedShareSequenceStart,
```
nit: maybe? 
Otherwise, the error wording is a bit weird to me, could be
```suggestion
    /// Share is not a start of a new blob.
    #[error("Share does not start a sequence")]
    NotSequenceStart,
```